commit fe4feb0641b2fce1d9e1a7e1c317b4222e0ff4f0
Author: Arseny V. Povolotsky <arseny.povolotsky@epfl.ch>
Date:   Fri Nov 9 11:44:29 2018 +0100

    iii1
    
    Change-Id: Iaabb4a28687148eea4a7734333784a02155b178f

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1db22f3..39c8958 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 set(LIBSONATA_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/include)
 
-set(LIBSONATA_COMPILE_OPTIONS -Wall -Wextra -pedantic -Werror)
+set(LIBSONATA_COMPILE_OPTIONS -Wall -Wextra -Werror)
 
 # =============================================================================
 # Dependencies
